Updated the containerd and cri doc (#5521)
DataDog/integrations-core: containerd/README.md

If you are using the Agent in a container, setting the `DD_CRI_SOCKET_PATH` environment variable to the Containerd socket automatically enables the `Containerd` Check with the default configuration.

> For example, to install the integration on Kubernetes, edit your daemonset to mount the Containerd socket from the host node to the agent container and set the `DD_CRI_SOCKET_PATH` env var to the daemonset mountPath:

Reviewer: ```suggestion
For example, to install the integration on Kubernetes, edit your daemonset to mount the Containerd socket from the host node to the Agent container and set the `DD_CRI_SOCKET_PATH` env var to the daemonset mountPath:
```

          volumeMounts:
            - name: containerdsocket
              mountPath: /var/run/containerd/containerd.sock
            - mountPath: /host/var/run
              name: var-run
              readOnly: true
          volumes:
            - hostPath:
                path: /var/run/containerd/containerd.sock
              name: containerdsocket
            - hostPath:
                path: /var/run

>               type: DirectoryOrCreate

Reviewer 2: we should remove `type: DirectoryOrCreate` as thisshould already exist on the host

Author: 👍

```

> Note: The `/var/run` directory needs to be mounted from the host or created, to run the integration without any issues.

Reviewer: ```suggestion
**Note**: The `/var/run` directory must be created or mounted from the host to run the integration without issues.
```
